### PR TITLE
Setup rpackage

### PR DESCRIPTION
**Pull request by dar19**

### CODE REVIEW

**Comment by leetaiyi** [comment]
Not ready to do this yet

File: R/diff_length_funcs.R [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    } else {
+        est_head = "meandiff"
+    }
+    tryCatch({   
+        if (test == "t") {
+            res = lm(as.formula(model), data = data_file_sub)
+            out = summary(res)$coefficients[2,c(1,4)]
+        } else if (test == "m") {
+            model = paste(model, "(1 | lib_id)", sep = "+")
+            res = lme4::lmer(as.formula(model), data = data_file_sub)
```
Comment (mnsmar) [inline R/diff_length_funcs.R:20]: `lme4` dependency? Needs to be imported here

File: scripts/diff_length.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                help="Normalize data by normalization group. Requires norm_group column in metadata where replicates have the same value with each other [default %default]"),
+    make_option(c("-o","--ofile"), default = "stdout",
+                help="Path to output file [default %default]")
+)
+opt <- parse_args(OptionParser(option_list = option_list))
+
+vars = c("data_path", "metadata_path","test","baseline","logscale","params", "norm", "ofile")
+for (i in 1:length(vars)) {
+    assign(vars[i],opt[[vars[i]]])
+}
```
Comment (mnsmar) [inline scripts/diff_length.R:32]: Why is this complicated code needed and you don't just use `opt$VAR` e.g. `opt$metadata_path` whenever needed? Seems like this is doing double the work (options need to be defined in 2 places).

File: scripts/nanoplen_two_col_plots.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+"""
+Pivot (convert table form long-to-wide format), log2-transform and create plots
```
Comment (mnsmar) [inline scripts/nanoplen_two_col_plots.py:4]: The long to wide are just implementation details that the user does not care. Just describe what the script offers.

**Pull request merged**